docs/tutorials/remote_soc.rst

```diff
@@ -0,0 +1,209 @@
+Building Your Own SoC
+=====================
+
+This tutorial will walk you through the process of building an ASIC containing one PicoRV32 RISC-V CPU core and 2 kilobytes of SRAM, on an open-source 130nm Skywater process node.
+
+You can find complete example designs which reflect the contents of this tutorial in the public SiliconCompiler repository, `with RAM <https://github.com/siliconcompiler/siliconcompiler/tree/main/examples/picorv32_ram>`_ and `without RAM <https://github.com/siliconcompiler/siliconcompiler/tree/main/examples/picorv32>`_.
+
+See the :ref:`Installation <Installation>` section for information on how to install SiliconCompiler, and the :ref:`Remote Processing <Remote Processing>` section for instructions on setting up the remote workflow.
+
+[TODO / CR feedback: Add a screenshot here (or at the bottom)?]
+
+Download PicoRV32 Verilog Code
+------------------------------
+
+The heart of any digital design is its HDL code, typically written in a language such as Verilog or VHDL. High-level synthesis languages are gaining in popularity, but most of them still output their final design sources in a traditional HDL such as Verilog.
+
+PicoRV32 is an open-source implementation of a small RISC-V CPU core, the sort you might find in a low-power microcontroller. Its source code, license, and various tooling can be found `in its GitHub repository <https://github.com/YosysHQ/picorv32>`_.
```

Comment:
Do we want the user to use our example (since the example without RAM is the same as what's here) or download from the YosysHQ GitHub repo that's pointed here?

Reply:
Personally, I find it helpful to have a complete example to refer to when I follow tutorials that try to teach by building a project from the ground up.

When a bunch of small code blocks are interspersed with explanations, it can sometimes be hard to follow where exactly each one is supposed to go in the partially-completed example. So I thought it might be helpful to link to known-good examples for the designs that the tutorial covers.

Reply:
Make sense. 
In that case, this will help make more clear this is the case, e.g, add:
> **This tutorial will walk you through downloading the design files from the source, but for your reference,** complete example designs which reflect the contents of this tutorial **are** in the public SiliconCompiler repository, [with RAM](https://github.com/siliconcompiler/siliconcompiler/tree/main/examples/picorv32_ram) and [without RAM](https://github.com/siliconcompiler/siliconcompiler/tree/main/examples/picorv32).

Reply:
I ended up rewriting the paragraph a bit:

> We will walk through the process of downloading the design files and writing a build script, but for your reference, you can find complete example designs which reflect the contents of this tutorial in the public SiliconCompiler repository. The first part of the tutorial will cover building the CPU core [without RAM], and the second part will describe how to [add an SRAM block].